src/components/CustomerQuota/ItemsSelection/Item.tsx

```diff
@@ -15,6 +16,56 @@ const styles = StyleSheet.create({
   },
 });
 
+/**
+ * Check if the campaign contains tt-token
+ * to indicate pod camapaign
+ *
+ * @param cartItemCategory category of item
+ */
+const isPodCampaign = (cartItemCategory: string): boolean =>
+  cartItemCategory.includes("tt-token");
+
+/**
+ * Check if pod is chargeable
+ *
+ * @param identifiers policy identifiers
+ * @param cartItem policy cart item
+ */
+const isPodChargeable = (
+  identifiers: PolicyIdentifier[],
+  cartItem: CartItem
+): boolean =>
+  identifiers.length > 0 &&
+  cartItem.category.includes("tt-token-lost") &&
+  cartItem.descriptionAlert === "*chargeable";
+
+/**
+ * Filters out the payment receipt identifier if not required
+ * which happens when pod is not chargeable
+ * Shape of identifiers and cartItem.identifierInputs are slightly different and
+ * hence require to filter separately
+ *
+ * @param newIdentifiers identifiers that can be modified
+ * @param newCartItem cartItem containing identifiers that can be modified
+ */
+const removePaymentReceiptField = (
+  newIdentifiers: PolicyIdentifier[],
```

Comment:
nitty-gritty but wdyt of the params to be `identifiers` and `cartItem` instead? 🤔

Reply:
good point, made the change! lemme know its this looks ok